README.md

```diff
@@ -64,6 +65,8 @@ Nous utilisons [11ty](https://www.11ty.dev/docs/) et [Netlify](https://docs.netl
 - Copiez le référentiel `git clone https://github.com/cds-snc/gcds-docs`.
 - Exécutez ensuite `npm install` pour installer toutes les dépendances Node.js.
 - Exécutez `npm start` pour démarrer un serveur Web avec rechargement à chaud.
+- [FR] The site will be accesible on http://localhost:8080/en or http://localhost:8080/fr. You have to append the `/en` and `/fr` as there is no page available at the base url
```

Comment:
Can you get this translated by Amy or Anik so that this whole section stays in French? You could also try the `traduction-gcds` channel.